controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -955,6 +957,13 @@ bool master_thread::handle_cmdu_1905_channel_preference_report(Socket *sd,
     return son_actions::send_cmdu_to_agent(sd, cmdu_tx);
 }
 
+bool master_thread::handle_cmdu_1905_ack_message(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    auto mid = cmdu_rx.getMessageId();
+    LOG(INFO) << "Received ACK_MESSAGE, mid=" << std::hex << int(mid);
```

Comment:
This should be `LOG(DEBUG)`

Reply:
Should add a `//TODO` comment which mentions that the ACK should be sent to the correct task and will be done as part of agent certification (nitpick)